plugin-server/src/utils/event.ts

```diff
@@ -0,0 +1,19 @@
+import { ProcessedPluginEvent } from '@posthog/plugin-scaffold'
+
+import { PreIngestionEvent } from '../types'
+
+export function convertPreIngestionEvent(event: PreIngestionEvent): ProcessedPluginEvent {
```

Comment:
nit:

Types tell the story here, but I'd rather have `convertToProcessedPluginEvent` or something.

Consider e.g.

`convertWater(water)` vs `convertToWine(water)`